source/feature-guide/feature-importing.rst

```diff
@@ -413,6 +427,12 @@ attribute's default value instead.
 An empty ``order`` for the dataset will be handled as if no order was
 passed in.
 
+An empty ``hidden`` for the dataset, will assume a flat order for all the
+variables that have `discarded: false` in their definitions.
+
+No variables can be in both the ``order`` and ``hidden`` member implicitly or
```

Comment:
I'm not sure what this sentence means, but you may already have said it elsewhere.

Reply:
I added it here because this is under the **Validation Rules** subtitle, so it is now one of the rules.

I will see to rephrase this.